src/IssueViz.Security.UnitTests/HotspotsList/HotspotsTableDataSourceTests.cs

```diff
@@ -289,7 +287,20 @@ private static HotspotsTableDataSource CreateTestSubject(Mock<ITableManager> tab
                 .Setup(x => x.GetTableManager(HotspotsTableConstants.TableManagerIdentifier))
                 .Returns(tableManagerMock.Object);
 
-            var testSubject = new HotspotsTableDataSource(tableManagerProviderMock.Object);
+            IAnalysisIssueVisualization requestedIssueViz = null;
+            var tableEntryFactoryMock = new Mock<IHotspotTableEntryFactory>();
+            tableEntryFactoryMock
+                .Setup(x => x.Create(It.IsAny<IAnalysisIssueVisualization>()))
+                .Callback((IAnalysisIssueVisualization issueViz) => requestedIssueViz = issueViz)
+                .Returns(() =>
```

Comment:
I didn't want to change the tests of this class, and they work fine as-is. So I'm mocking the factory to just return a table entry that has the given IssueViz as Identity.